### PR TITLE
Update adguard from 1.5.14 to 2.0.4

### PR DESCRIPTION
**Pull request by SharjeelJ**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
> It seems the App source '/usr/local/Caskroom/adguard/2.0.4/Adguard.app' is not there.

How to fix: [Reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs) → [`source is not there` error](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/source_is_not_there_error.md).

**Comment by SharjeelJ** [comment]
I Believe I fixed it. The original was just a .app in the root of the .dmg whereas the new format is a .pkg in the root of the .dmg (adjusted from app -> pkg). I also added in the "uninstall pkgutil" line with the correct pkg identifier. All in all, let me know if anything I did is done wrong / could be improved so I can do so in the future :)

File: Casks/adguard.rb [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
   url "https://static.adguard.com/mac/Adguard-#{version}.release.dmg"
   appcast 'https://static.adguard.com/mac/adguard-release-appcast.xml'
   name 'Adguard for Mac'
   homepage 'https://adguard.com/'
 
-  app 'Adguard.app'
+  pkg 'AdGuard.pkg'
+
+  uninstall pkgutil: [
```
Comment (vitorgalvao) [inline Casks/adguard.rb]: Since it’s only one element, it can be a string. I.e. `uninstall pkgutil: 'com.adguard.mac.adguard-pkg'`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/adguard.rb (hunk 1 above)
Comment (SharjeelJ) [reply Casks/adguard.rb]: Adjusted it, thank you for the tip :)

**Comment by vitorgalvao** [comment]
Great job! Thank you. We’ll be able to merge this as soon as Travis CI passes (again).

**Pull request merged**